Form/Type/FacebookType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\SocialBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Jan Sanne Mulder <jansanne@e-active.nl>
+ */
+class FacebookType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add('user_id', 'text', ['attr' => ['readonly' => 'true']]);
+        $builder->add('access_token', 'text', ['attr' => ['readonly' => 'true']]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
```

Comment:
As of symfony 2.8 (which is what 0.7 uses) the getName method is deprecated as the class name is now used as form name. This also means that the form name in the configuration should be updated with the class names.

It is also recommanded to define a getBlockPrefix method to set a template name. For which the current form name can be used. So just rename getName to getBlockPrefix and your golden.